README.md

```diff
@@ -8,8 +8,8 @@
 Tax-Calculator
 ==============
 
-Tax-Calculator is an open-source microsimulation model for static
-analysis of USA federal income and payroll taxes.
+Tax-Calculator is an open-source microsimulation model for analysis of
```

Comment:
@martinholmer Why the removal of the adjective "static" here?

Reply:
That's still a "static" score in the parlance of tax scoring (since it's still fixed GNP assumption -- as opposed to dynamic scoring where GDP varies).

Reply:
@jdebacker said:
> That's still a "static" score in the parlance of tax scoring (since it's still fixed GNP assumption -- as opposed to dynamic scoring where GDP varies).

Can you provide a reference that spells out the definition of "static" and "dynamic"?
Where do "behavioral responses" fit into the "static" vs "dynamic" distinction?

Reply:
https://taxfoundation.org/taxedu/glossary/conventional-scoring-static-scoring/

Reply:
Thanks for the reference.  See commit [92fc3db](https://github.com/PSLmodels/Tax-Calculator/pull/2982/commits/92fc3db3536b62c47aa86b5c4be1633ff8ea8cd0) for the "conventional analysis" terminology now used in `README.md`.